ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -82,6 +73,13 @@ export const CromwellConfigurationPanel = ({
     setTimeout(() => setSidebarActiveIconStore.next('apps'), 3000);
   };
 
+  const cromwellConfig: CreateAppRequest = {
+    ...defaultCromwellConfig,
+    persistentDiskRequest:
+      disk !== undefined ? disk : defaultCromwellConfig.persistentDiskRequest,
```

Comment:
This is the core change (in addition to the equivalent RStudio change); the remainder is mostly refactors.